src/utils/decisions.ts

```diff
@@ -0,0 +1,22 @@
+import { DecisionDialogValues } from '~common/ColonyDecisions/DecisionDialog/DecisionDialog';
+import { Address, Decision } from '~types';
+
+const getLocalStorageDecisionKey = (walletAddress: Address) =>
+  `decision:${walletAddress}`;
```

Comment:
Having a key that contains the wallet address saves us having to implement logic to detect whether the current user is the same user as the user that stored the decision. We will just save one decision per user, and retreive that decision if the user wants to make an edit to their decision in decision preview.